### ISSUE TITLE
Comparison experiment using BCI dataset

### ISSUE THREAD

**Issue by msalvaris** [Azure/fast_retraining#19]
- [x] Download dataset
- [x] Create data loader and preprocessing
- [x] Create initial xgboost pipeline
- [x] Create initial lightgbm pipeline
- [x] Create comparison on test set
- [x] Add docs
